packages/manager/modules/nasha/src/dashboard/general-information/general-information.template.html

```diff
@@ -0,0 +1,97 @@
+                <!-- ID -->
+                <oui-tile-definition
+                    data-term="{{:: ('nasha_dashboard_general_information_information_id' | translate) }}"
+                    data-description="{{:: $ctrl.nasha.serviceName }}"
+                >
+                </oui-tile-definition>
+                <!-- Region -->
+                <oui-tile-definition
+                    data-term="{{:: ('nasha_dashboard_general_information_information_datacenter' | translate) }}"
+                    data-description="{{:: $ctrl.nasha.localeDatacenter }}"
+                >
+                </oui-tile-definition>
+                <!-- Format -->
+                <oui-tile-definition
+                    data-term="{{:: ('nasha_dashboard_general_information_information_disk_type' | translate) }}"
+                    data-description="{{:: $ctrl.nasha.diskType }}"
+                >
+                </oui-tile-definition>
+                <!-- Size -->
+                <oui-tile-definition
+                    data-term="{{:: ('nasha_dashboard_general_information_information_disk_size' | translate) }}"
+                    data-description="{{:: $ctrl.nasha.diskSize }}"
+                >
+                </oui-tile-definition>
+            </oui-tile>
+        </div>
+        <!-- Configuration -->
+        <div class="col-md-4">
+            <oui-tile
+                data-heading="{{:: ('nasha_dashboard_general_information_configuration_title' | translate) }}"
+            >
+                <!-- Quota -->
+                <oui-tile-definition
+                    data-term="{{:: ('nasha_dashboard_general_information_configuration_quota' | translate) }}"
+                >
+                    <oui-tile-description>
+                        <nasha-components-space-meter
+                            class="cloud-space-meter"
+                            data-usage="$ctrl.nasha.use"
+                            data-large="true"
+                        >
+                        </nasha-components-space-meter>
+                        <oui-button
+                            size="m"
+                            variant="link"
+                            icon-right="oui-icon-arrow-right"
+                        >
+                            <span
+                                data-translate="nasha_dashboard_general_information_configuration_link"
+                            ></span>
+                        </oui-button>
+                    </oui-tile-description>
+                </oui-tile-definition>
+            </oui-tile>
+        </div>
+        <div class="col-md-4">
+            <billing-subscription-tile
+                on-error="$ctrl.onBillingInformationError(error)"
+                go-to-commit="$ctrl.goToCommit()"
+                go-to-cancel-commit="$ctrl.goToCancelCommit()"
+                go-to-cancel-resiliation="$ctrl.goToCancelResiliation()"
+                go-to-resiliation="$ctrl.goToResiliation()"
```

Comment:
Already present in the billing-subscription-tile module except if you need to customize them